tutorials/vbpr_text.ipynb

```diff
@@ -0,0 +1,240 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "*Copyright (c) Cornac Authors. All rights reserved.*\n",
+    "\n",
+    "*Licensed under the Apache 2.0 License.*\n",
+    "\n",
+    "# Visual Bayesian Personalizer Ranking (VBPR) with Text Data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<table class=\"tfo-notebook-buttons\" align=\"left\">\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/PreferredAI/cornac/blob/master/tutorials/vbpr_text.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Run in Google Colab</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://github.com/PreferredAI/cornac/blob/master/tutorials/vbpr_text.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />View source on GitHub</a>\n",
+    "  </td>\n",
+    "</table>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Overview\n",
+    "In this tutorial, we will employ a recommendation algorithm designed to leverage item images to work with item text content.\n",
```

Comment:
It seems like the first two paragraphs are closely related, and it could be better if they can be summarized into one. In particular, the question regarding the availability of image data can come earlier. I suggest something like this: "We are interested in the following situation: we would like to use the Visual Bayesian Personalized Ranking (VBPR) model, which leverages pre-trained item visual features to improve recommendation. However, our data of interest, namely MovieLens, does not come with visual information, but instead it contains text movie plots.  In this tutorial, we will see how Conac's *modality* infrastructures allow us to easily use VBPR (originally designed for visual information) to leverage item text content." If you choose to adopt this suggestion, my comments below on the first two paragraphs become useless, so please feel free to ignore them.

Reply:
updated